packages/components/image-viewer/src/image-viewer.vue

```diff
@@ -3,50 +3,53 @@
     <div
       ref="wrapper"
       :tabindex="-1"
-      class="el-image-viewer__wrapper"
+      :class="ns.e('wrapper')"
       :style="{ zIndex }"
     >
-      <div
-        class="el-image-viewer__mask"
-        @click.self="hideOnClickModal && hide()"
-      />
+      <div :class="ns.e('mask')" @click.self="hideOnClickModal && hide()" />
 
       <!-- CLOSE -->
-      <span class="el-image-viewer__btn el-image-viewer__close" @click="hide">
+      <span :class="[ns.e('btn'), ns.e('close')]" @click="hide">
         <el-icon><close /></el-icon>
       </span>
 
       <!-- ARROW -->
       <template v-if="!isSingle">
         <span
-          class="el-image-viewer__btn el-image-viewer__prev"
-          :class="{ 'is-disabled': !infinite && isFirst }"
+          :class="[
+            ns.e('btn'),
+            ns.e('prev'),
+            { [ns.is('disabled')]: !infinite && isFirst },
```

Comment:
ns.is('disabled',  !infinite && isFirst)

Reply:
Done.